js/okex.js

```diff
@@ -3176,6 +3177,34 @@ module.exports = class okex extends Exchange {
         return this.parseFundingRate (entry, market);
     }
 
+    async fetchFundingHistory (symbol = undefined, since = undefined, limit = undefined, params = {}) {
+        await this.loadMarkets ();
+        const request = {
+            'type': 8,
+        };
+        if (limit !== undefined) {
+            request['limit'] = limit;
+        }
+        const response = await this.privateGetAccountBills (this.extend (request, params));
+        const data = this.safeValue (response, 'data');
+        const result = [];
+        for (let i = 0; i < data.length; i++) {
+            const entry = data[i];
+            const timestamp = this.safeTimestamp (entry, 'ts');
+            const instId = this.safeString (entry, 'instId');
+            result.push ({
+                'info': entry,
+                'symbol': this.safeSymbol (instId),
+                'code': this.safeCurrencyCode (instId),
```

Comment:
I don't think this will work. Instead you should do this:

```
const market = this.safeMarket (instId);
const code = market['inverse'] ? market['base'] : market['quote'];
const symbol = market['symbol'];
```